load-tests/in_memory_hub/deployment/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
+  - in_memory_hub.yaml
```

Comment:
Currently c2c doesn't support websubhub listener generation, therefore it does not generate required k8s services. We need to fix this from c2c cide. https://github.com/ballerina-platform/module-ballerina-c2c/issues/483
Till its fixed, We can generate the service using Kustomize. 
You can add following contents.

Kustomizations.yaml
```
resources:
  - in_memory_hub.yaml
  - service.yaml

patches:
  - path: deployment-patch.yaml
    target:
      group: apps
      version: v1
      kind: Deployment
      name: in-memory-hub-deployment
```

deployment-patch.yaml
```yaml
apiVersion: "apps/v1"
kind: Deployment
metadata:
  name: no-name
spec:
  template:
    metadata:
      labels:
        logs: "true"
    spec:
      containers:
        - name: "in-memory-hub-deployment"
          imagePullPolicy: Always
          ports:
            - containerPort: 9090
              name: "port-1-in-mem"
              protocol: "TCP"
```
service.yaml
```yaml
---
apiVersion: "v1"
kind: "Service"
metadata:
  labels:
    app: "in_memory_hub"
  name: "in-memory-hub-svc"
spec:
  ports:
    - name: "port-1-in-mem"
      port: 9090
      protocol: "TCP"
      targetPort: 9090
  selector:
    app: "in_memory_hub"
  type: "ClusterIP"

```

Reply:
Fixed with [057ee41615f5c62a7aa8ceb2b0e58f60866d582d](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/432/commits/057ee41615f5c62a7aa8ceb2b0e58f60866d582d)